examples/burgers1d.ipynb

```diff
@@ -118,175 +117,78 @@
     "\n",
     "autoencoder.load_state_dict(autoencoder_param)\n",
     "\n",
-    "# Z = autoencoder.encoder(X_train).detach().numpy()\n",
-    "Z = autoencoder.encoder(X_train)\n",
-    "Z0 = initial_condition_latent(param_grid, physics, autoencoder)\n",
-    "Zis_samples, gp_dictionnary, interpolation_data, sindy_coef, n_coef, coef_samples = simulate_interpolated_sindy(param_grid, Z0, t_grid, n_samples, Dt, Z, param_train, fd_type)\n",
-    "Zis_mean, _, _, _, _, _ = simulate_interpolated_sindy_mean(param_grid, Z0, t_grid, Dt, Z, param_train, fd_type)\n",
+    "n_coef = sindy.ncoefs\n",
+    "\n",
+    "from lasdi.gplasdi import sample_roms, average_rom\n",
+    "Zis_samples = sample_roms(autoencoder, physics, sindy, gp_dictionnary, param_grid, n_samples)\n",
+    "Zis_mean = average_rom(autoencoder, physics, sindy, gp_dictionnary, param_grid)\n",
+    "\n",
+    "print(Zis_mean.shape)\n",
+    "print(Zis_samples.shape)\n",
+    "\n",
+    "X_pred_mean = autoencoder.decoder(torch.Tensor(Zis_mean)).detach().numpy()\n",
+    "X_pred_samples = autoencoder.decoder(torch.Tensor(Zis_samples)).detach().numpy()\n",
+    "\n",
+    "avg_rel_error = np.zeros(param_grid.shape[0])\n",
+    "for k in range(param_grid.shape[0]):\n",
+    "    avg_rel_error[k], _ = compute_errors(X_pred_mean[k], physics, X_test[k].numpy())\n",
     "\n",
-    "_, _, max_e_relative_mean, _ = compute_errors(n_a_grid, n_w_grid, Zis_mean, autoencoder, X_test, Dt, Dx)\n",
-    "_, _, _, max_std = compute_errors(n_a_grid, n_w_grid, Zis_samples, autoencoder, X_test, Dt, Dx)\n",
+    "max_std = np.zeros(param_grid.shape[0])\n",
+    "for k in range(param_grid.shape[0]):\n",
+    "    max_std[k] = X_pred_samples[k].std(0).max()\n",
     "\n",
-    "gp_pred = eval_gp(gp_dictionnary, param_grid, n_coef)"
+    "avg_rel_error = avg_rel_error.reshape([n_w_grid, n_a_grid]).T\n",
+    "max_std = max_std.reshape([n_w_grid, n_a_grid]).T\n",
+    "\n",
+    "print(avg_rel_error.shape)\n",
+    "print(max_std.shape)\n",
+    "\n",
+    "from lasdi.interp import eval_gp\n",
```

Comment:
Should this be : `from lasdi.gp import eval_gp`  instead of `from lasdi.interp import eval_gp` ?

Reply:
This is fixed now.